DBN_article/DBN_article.md

```diff
@@ -0,0 +1,187 @@
+entire network parameters, adjusting them to minimize the loss, 
+optimize the gradient descent, adjusting weights and biases 
+based on the gradients of the loss with respect to the parameters 
+(you may perform this step in an iterative manner);
+7. ~~word-by-word, say loudly and clearly the spell;~~
+8. evaluate the trained DBN on a validation set;
+9. postprocessing – you might add some steps as tresholding or 
+normalization.
+
+<div style="text-align: center">
+    <img src="dbn_5.png" alt="RBM training" style="width:500px;"/>
+</div>
+
+## Use Cases
+
+&nbsp;&nbsp;&nbsp;On broad terms, Deep Belief Networks can be described as more 
+efficient version of feedforward neural network. There follows a 
+vast applicability of this type of networks: image recognition, 
+recognizing, clustering and generating of images, video sequences 
+and motion-caption data. For example, ConvDBNs are good to use 
+for tasks that involve grid-like data, such as images, while 
+Temporal DBNs are better for speech recognition or natural 
+language processing tasks. To sum up, the fields they are widely 
+used are:
+
+* Computer Vision – object recognition and classification;
+* NLP – sentiment analysis and text classification;
+* Speech recognition -  transcribing speech into text;
+* Recommendation Systems - giving suggestions based on previous inputs;
+* Financial analysis – predicting stock market and how risky some actions are;
+* Bioinformatics – predict the interactions between components like proteins, 
+find new drugs and predict how genes may express.
+
+## Drawbacks and limits
+
+&nbsp;&nbsp;&nbsp;In spite of a lot advantages DBNs brings, they are still a quite 
+early version of a deep neural network and might be not so 
+effective as its newest aliases due to a considerable set of 
+drawbacks, such as:
+
+* high hardware requirements;
+* complex data model that is difficult to train;
+* difficult to use by unexperienced people;
+* requires of a huge amount of data for a good performance;
+* requires classifiers to grasp the output.
+
+## Conclusion
+
+&nbsp;&nbsp;&nbsp;To summarize everything up, Deep Belief Network is an 
+early-deep-learning-days architecture, composed of multiple 
+Restricted Boltzmann Machines, aimed to perform classification, 
+clustering and generation tasks. Due to the presence of RBMs, 
+trained once at a time, it uses Greedy Algorithm to train each 
+layer until all RBM are trained and the output can be passed to 
+supervised learning model. Although it is quite old, it still 
+has its fields of applicability where it performs considerable 
+better than other known algorithms, that being one of the reasons 
+to know about its existence. Not the last, it is just an awesome 
+algorithm with a unique architecture that might bring you fun 
+while diving in!
+
+<div style="text-align: center">
+    <img src="dbn_6.jpg" alt="Closing image" style="width:500px;"/>
+</div>
+
+
+## References
```

Comment:
I would love to see more resources that you utilized or maybe that you could also add